test/events/component-view.test.js

```diff
@@ -0,0 +1,226 @@
+/*global require, describe, it, before, after, sinon, document */
+require('../setup');
+const assert = require('assert');
+const settings = require('../../src/javascript/core/settings');
+const send = require('../../src/javascript/core/send');
+const core = require('../../src/javascript/core');
+const session = require('../../src/javascript/core/session');
+const componentView = require('../../src/javascript/events/component-view');
+
+const config = {
+	context: {
+		product: 'desktop'
+	},
+	user: {
+		user_id: '123456'
+	},
+};
+
+let targetComponent;
+let error;
```

Comment:
I think this is good as it is, but do you think this would be marginally clearer if it was called `errorThrown`?

Reply:
I agree!